rs/sns/governance/src/extensions.rs

```diff
@@ -56,6 +58,51 @@ thread_local! {
     }});
 }
 
+#[cfg(any(test, feature = "test"))]
+lazy_static! {
+    static ref TEST_HASH_TO_EXTENSION_SPEC: BTreeMap<[u8; 32], ExtensionSpec> = {
```

Comment:
This replaces create_test_allowed_extensions, and so even though it looks like this chunk is all new code, most of it is actually just moved from lines 1649-1678. There is some new code here though, namely, smallest_valid_wasm_hash.